test/test_check_db_dump_recent.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python3
+
+"""
+This file tests the script that checks the GOCDB failover process is
+happening.
+"""
+
+from datetime import datetime, timedelta
+import os
+import tempfile
+import unittest
+
+# Need to import the module as a whole to monkey patch the underlying
+# datetime.now() calls.
+import check.check_db_dump_recent as check_db_dump_recent
```

Comment:
```suggestion
from check import check_db_dump_recent
```
Pretty sure you can do `from package import module` which would be more standard Python.

Reply:
Well, not so much standard, but clearer as you're not actually renaming the module, just shortening calls to it.

Reply:
Addressed in 0ff8f3c515c42c4e75aeb6803d42e1ec24f5e568